index.js

```diff
@@ -0,0 +1,515 @@
+document.addEventListener("DOMContentLoaded", () => {
+
+
+    // Question 1
+    const changeHead = () => {
+        let str = document.querySelector("#string").value;
+        document.querySelector("#head1").innerText = str;
+    }
+
+    let submit = document.querySelector("#changeHeadingBtn");
+    submit.addEventListener("click", ()=> {
```

Comment:
I think you already know this but you can simply add 

`submit.addEventListener("click", changeHead)`

The event listener invokes the function when the event happens.